src/components/Popover/PopoverList.css

```diff
@@ -0,0 +1,10 @@
+::-webkit-scrollbar {
```

Comment:
Can you rename this to `PopoverList.module.css` so we know it's a CSS module.

Also doesn't `Popover` component need this too as it can have a lot of content

Reply:
@rushi  Can it be PopoverScroll.module.css as I also included it in Popover component in this revision?